api/src/main/proto/stellarstation/api/v1/antenna/antenna.proto

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package stellarstation.api.v1.antenna;
+
+option go_package = "github.com/infostellarinc/go-stellarstation/api/v1/antenna";
+
+option java_multiple_files = true;
+option java_outer_classname = "AntennaProto";
+option java_package = "com.stellarstation.api.v1.antenna";
+
+// Messages for modeling physical antenna configuration in the API.
+
+// The static configuration of an antenna. These are the physical parameters of the antenna.
+message AntennaConfiguration {
+
+  // The latitude of the antenna, in degrees.
+  double latitude = 1;
+
+  // The longitude of the antenna, in degrees.
+  double longitude = 2;
+
+  // The altitude of the antenna, in meters.
```

Comment:
Please note that altitude is above mean sea level. 
@rtubio Could you suggest me the necessity to note referenced world geodetic system model to the coordination when we control ACU for S/X-band directly based on trajectory calculated by StellarStation?